evm2wast.c

```diff
@@ -0,0 +1,72 @@
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <dirent.h>
+#include <math.h>
+#include "evm2wast.h"
+#include "gadgets.h"
+#include "util.h"
+#define digits(x) (floor(log10(abs(x))) + 1) 
```

Comment:
I like inline functions better because type matching is worse with such complex macros.

For example I get this:
```
warning: taking the absolute value of unsigned type 'size_t' (aka 'unsigned long') has no effect [-Wabsolute-value]
note: remove the call to 'abs' since unsigned values cannot be negative
```